omero/sysadmins/whatsnew.txt

```diff
@@ -1,14 +1,11 @@
 What's new for OMERO 5.1
 ========================
 
-From a sysadmin's point of view, here is some introductory / overview
-text.
-
-- :doc:`server-upgrade` has **not yet** been updated to explain the
-  server upgrade process from OMERO 5.0 to OMERO 5.1.
-
-- :doc:`system-requirements` has been updated to reflect **various
-  things yet to be finalized**.
+- :doc:`version-requirements` provides extensive details about which operating
+  systems and dependency versions we will be supporting for the life of 5.1
+  and the likely changes to these for the next major release (currently slated
+  to be 5.2). :doc:`system-requirements` has been updated to reflect these
+  updates.
```

Comment:
The system requirements are not solely based upon the version requirements though.  There's the question of how much we can realistically support and for how long.  In this case we've chosen to support the last two LTS releases.  While 10.04 will still work, it's in the "use at own risk" category: we have done *zero* building, testing and QA on this platform for 5.1. And support for the 5.1 deployment timeframe is also considered; for 10.04 LTS, support ends in April next month, making it unrealistic for a 5.1 deployment.